package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@ptomasroos/react-native-multi-slider",
-  "version": "0.0.8",
+  "version": "1.0.0",
```

Comment:
Bumping version numbers should not be part of the PR will be done with a script later